Client.xcodeproj/project.pbxproj

```diff
@@ -49,6 +49,20 @@
 		0BF1B7E31AC60DEA00A7B407 /* InsetButton.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0BF1B7E21AC60DEA00A7B407 /* InsetButton.swift */; };
 		0BF42D4F1A7CD09600889E28 /* TestFavicons.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0BF42D4E1A7CD09600889E28 /* TestFavicons.swift */; };
 		0BF8F8DA1AEFF1C900E90BC2 /* noTitle.html in Resources */ = {isa = PBXBuildFile; fileRef = 0BF8F8D91AEFF1C900E90BC2 /* noTitle.html */; };
+		1B308A9F2714F01E001D6200 /* SwiftKeychainWrapper in Frameworks */ = {isa = PBXBuildFile; productRef = 1B3D99F7270E8CF8006E1264 /* SwiftKeychainWrapper */; };
+		1B308AA02714F01E001D6200 /* SwiftyJSON in Frameworks */ = {isa = PBXBuildFile; productRef = 1BDA1F8627113817006B2FAB /* SwiftyJSON */; };
+		1B308AA12714F01E001D6200 /* SwiftProtobuf in Frameworks */ = {isa = PBXBuildFile; productRef = 1BDA1F8B271138A5006B2FAB /* SwiftProtobuf */; };
+		1B3D99F1270E89D0006E1264 /* Telemetry in Frameworks */ = {isa = PBXBuildFile; productRef = 1B3D99F0270E89D0006E1264 /* Telemetry */; };
+		1B750BE1274312D3002CBA18 /* RustLog in Frameworks */ = {isa = PBXBuildFile; productRef = 1B750BE0274312D3002CBA18 /* RustLog */; };
+		1BE9ECB0271765C800D729BB /* Nimbus in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECAF271765C800D729BB /* Nimbus */; };
+		1BE9ECBE2717663100D729BB /* FxAClient in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECBD2717663100D729BB /* FxAClient */; };
+		1BE9ECC02717663200D729BB /* Logins in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECBF2717663200D729BB /* Logins */; };
+		1BE9ECC22717663200D729BB /* Places in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECC12717663200D729BB /* Places */; };
+		1BE9ECC42717663200D729BB /* Viaduct in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECC32717663200D729BB /* Viaduct */; };
+		1BE9ECCA27176FEA00D729BB /* SwiftyJSON in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECC927176FEA00D729BB /* SwiftyJSON */; };
+		1BE9ECCC2717704300D729BB /* SwiftyJSON in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECCB2717704300D729BB /* SwiftyJSON */; };
+		1BE9ECCE271770A600D729BB /* SwiftyJSON in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECCD271770A600D729BB /* SwiftyJSON */; };
+		1BE9ECD0271770AB00D729BB /* SwiftKeychainWrapper in Frameworks */ = {isa = PBXBuildFile; productRef = 1BE9ECCF271770AB00D729BB /* SwiftKeychainWrapper */; };
```

Comment:
I noticed that there's 8 modules from AS out of 10 I see in `Package.swift` [from here](https://github.com/mozilla/rust-components-swift/blob/main/Package.swift). Are the other two not ready to be consumed by iOS?

Reply:
iOS doesn't current use the application-services versions of Push, Tabs, Autofill and CrashTest (not an actual component) so those won't be added until there is a roadmap plan to have iOS switch over to using our versions!